//package com.sincetimes.redwar.game.comnunicate.protocol;


message RceObtainBattleReplay
{

}

